src/prefect/cli/task_run.py

```diff
@@ -52,7 +62,17 @@ async def inspect(id: UUID):
         except ObjectNotFound:
             exit_with_error(f"Task run '{id}' not found!")
 
-    app.console.print(Pretty(task_run))
+    if web:
+        task_run_url = url_for("task-run", obj_id=id)
+        if not task_run_url:
+            exit_with_error(
+                "Failed to generate URL for task run. Make sure PREFECT_UI_URL is configured."
```

Comment:
doesn't need to be addressed, just a comment: i would expect the `url_for` utility to raise this error instead of return `None`